modules/apps/organizations/organizations-service/src/main/java/com/liferay/organizations/internal/object/system/OrganizationSystemObjectDefinitionManager.java

```diff
@@ -44,7 +44,7 @@
 /**
  * @author Mateus Santana
  */
-@Component(enabled = false, service = SystemObjectDefinitionManager.class)
+@Component(service = SystemObjectDefinitionManager.class)
```

Comment:
There is a [PR](https://github.com/liferay-objects/liferay-portal/pull/2310) removing this Feature Flag, you can cherry-pick it.

Reply:
I removed this and added the @ignore in the test.